core/src/main/scala/com.snowplowanalytics.stream/loader/Config.scala

```diff
@@ -278,4 +280,7 @@ object Config {
       case cn @ "cn-north-1" => s"https://kinesis.$cn.amazonaws.com.cn"
       case _                 => s"https://kinesis.$region.amazonaws.com"
     })
+
+  // Used as an option prefix when reading system properties.
+  val Namespace = "esloader"
```

Comment:
I slightly prefer the namespace "snowplow" which is [what we used in postgres loader](https://github.com/snowplow-incubator/snowplow-postgres-loader/blob/70e7990548af033c02918a47ea3925ecc783f4f8/modules/loader/src/main/scala/com/snowplowanalytics/snowplow/postgres/config/Cli.scala#L118).  I know for sqs2kinesis we used something different, but that was because sqs2kinesis was supposed to be a general app, not just for snowplow.